### PR TITLE
Add import support to  google_dns_record_set

### LINKED ISSUE #743: Add import cmd  for google_dns_record_set
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Terraform v0.10.8

### Affected Resource(s)
Please list the resources as a list, for example:
- google_dns_record_set

### Expected Behavior
I can import  google_dns_record_set on my tfstate

### Actual Behavior
Import is not supported for resource: google_dns_record_set

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform import google_dns_record_set xxxx`

### PR DESCRIPTION
**Pull request by rosbo**
Fixes #743

### CODE REVIEW

**Comment by ean** [comment]
I tried out this PR, and it works great.
One small issue with an error message that is a bit confusing. The ID provided says it expects a different format than is really does.

```
$ terraform import google_dns_record_set.root-txt data
google_dns_record_set.root-txt: Importing from ID "data"...

Error: google_dns_record_set.root-txt (import id: data): import google_dns_record_set.root-txt (id: data): Invalid dns record specifier. Expecting {project}/{zone name}/{record name}/{record type}

```
```
$ terraform import google_dns_record_set.root-txt foo/zone-name/example.com./TXT
google_dns_record_set.root-txt: Importing from ID "foo/zone-name/example.com./TXT"...

Error: google_dns_record_set.root-txt (import id: foo/zone-name/example.com./TXT): import google_dns_record_set.root-txt (id: foo/zone-name/example.com./TXT): Invalid dns record specifier. Expecting {project}/{zone name}/{record name}/{record type}
```

From the source code https://github.com/terraform-providers/terraform-provider-google/pull/895/files#diff-8bfe0d9a51bbbdf68e796b35fac038ffR297 the expected format is `zone/name/type`

**Comment by rosbo** [comment]
I fixed the error message and improved the docs slightly.

File: google/resource_dns_record_set.go [hunk 1]
```diff
@@ -286,6 +293,19 @@ func resourceDnsRecordSetUpdate(d *schema.ResourceData, meta interface{}) error
 	return resourceDnsRecordSetRead(d, meta)
 }
 
+func resourceDnsRecordSetImportState(d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData, error) {
+	parts := strings.Split(d.Id(), "/")
+	if len(parts) != 3 {
+		return nil, fmt.Errorf("Invalid dns record specifier. Expecting {zone-name}/{record-name}/{record-type}")
```
Comment (ndmckinley) [inline google/resource_dns_record_set.go]: I think most people aren't going to know to add an extra . after the record name - might be a good place to tell them.

Comment (rosbo) [reply google/resource_dns_record_set.go]: Good idea. I also updated the docs to add a note about this.

**Pull request merged**